src/services/doppler-system-usage-api-client.ts

```diff
@@ -0,0 +1,44 @@
+import { HttpBaseClient } from './http-base-client';
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+
+export interface SystemUsage {
+  email: string;
+  reportsSectionLastVisit: string;
+}
+
+export interface DopplerSystemUsageApiClient {
+  getUser(): Promise<ResultWithoutExpectedErrors<SystemUsage>>;
+}
+
+export class HttpDopplerSystemUsageApiClient
+  extends HttpBaseClient
+  implements DopplerSystemUsageApiClient
+{
+  private clientName = 'System Usage';
+
+  public async getUser(): Promise<ResultWithoutExpectedErrors<SystemUsage>> {
+    try {
+      const { email, jwtToken } = this.getApiConnectionData(this.clientName);
```

Comment:
What is the meaning of getApiConnectionData's parameter?

Reply:
This parameter is only used to print who was the client that generated an error ... 
you can see its usage in src/services/http-bas
<img width="523" alt="Captura de Pantalla 2022-10-28 a la(s) 9 03 06 a  m" src="https://user-images.githubusercontent.com/84402180/198630803-a917b860-863e-44ee-8555-0ea8778beb07.png">
e-client.ts

Reply:
👍